modules/handlers/anonym_comment.py

```diff
@@ -16,5 +18,7 @@ def anonymous_comment_msg(update: Update, context: CallbackContext):
     info = EventInfo.from_message(update, context)
 
     if info.chat_id == config_map['meme']['channel_group_id'] and info.message.via_bot is None:
-        info.message.copy(chat_id=info.chat_id, reply_to_message_id=info.message.reply_to_message.message_id)
+        info.bot.send_message(chat_id=info.chat_id, 
```

Comment:
The issue with this approach is that only works with text messages. If the message contains an image or a file, it won't work, for the "text" property is not present. 
You would have to use the "caption" property instead, if present in that message type.